src/environmentd/src/http.rs

```diff
@@ -709,33 +729,57 @@ async fn init_ws(
             }
         }
     };
-    let (creds, options) = if frontegg.is_some() {
-        match ws_auth {
+    let (user, options) = match (frontegg.as_ref(), existing_user, ws_auth) {
```

Comment:
@RobinClowers all frontegg involvement is going away? really interested in that, please tag me when you put that code up